src/utilities/gTag.utils.ts

```diff
@@ -0,0 +1,3 @@
+export const gTagEvent = (eventName: string, eventProps: object) => {
```

Comment:
eventParams: object or  eventOptions: object
```suggestion
export const gTagEvent = (eventName: string, eventOptions: object) => {
```

Reply:
renamedd to eventParams